src/jit/lclvars.cpp

```diff
@@ -1727,12 +1752,27 @@ void Compiler::lvaPromoteStructVar(unsigned lclNum, lvaStructPromotionInfo* Stru
             fieldVarDsc->lvIsRegArg = true;
             fieldVarDsc->lvArgReg   = varDsc->lvArgReg;
             fieldVarDsc->setPrefReg(varDsc->lvArgReg, this); // Set the preferred register
+#if FEATURE_MULTIREG_ARGS && defined(FEATURE_SIMD)
+            if (varTypeIsSIMD(fieldVarDsc))
+            {
+                fieldVarDsc->lvOtherArgReg = varDsc->lvOtherArgReg;
```

Comment:
I didn't think that we really needed lvOtherReg anymore.
Can you add a comment explaining your usage of it here.